cmd/soroban-cli/src/cli.rs

```diff
@@ -0,0 +1,76 @@
+use clap::CommandFactory;
+use dotenvy::dotenv;
+use tracing_subscriber::{fmt, EnvFilter};
+
+use crate::{commands, Root};
+
+#[tokio::main]
+pub async fn main() {
+    let _ = dotenv().unwrap_or_default();
+
+    // Map SOROBAN_ env vars to STELLAR_ env vars for backwards compatibility
+    // with the soroban-cli prior to when the stellar-cli was released.
+    let vars = &[
+        "FEE",
+        "NO_CACHE",
+        "ACCOUNT",
+        "CONTRACT_ID",
+        "INVOKE_VIEW",
+        "RPC_URL",
+        "NETWORK_PASSPHRASE",
+        "NETWORK",
+        "PORT",
+        "SECRET_KEY",
+        "CONFIG_HOME",
+    ];
+    for var in vars {
+        let soroban_key = format!("SOROBAN_{var}");
+        let stellar_key = format!("STELLAR_{var}");
+        if let Ok(val) = std::env::var(soroban_key) {
+            std::env::set_var(stellar_key, val);
+        }
+    }
```

Comment:
This new file is really just a move of the old `main.rs` so it can be shared. The only new code is these lines that allow us to continue to support the SOROBNA_ env vars by mapping them to the STELLAR_ env vars.